src/components/toast/components/toast-display/toast-display.module.css

```diff
@@ -1,84 +1,84 @@
 .root {
-  border-radius: 6px;
-  border: 1px solid;
-  box-shadow: var(--token-elevation-higher-box-shadow);
-  display: flex;
-  padding: 16px;
+	border-radius: 6px;
+	border: 1px solid;
+	box-shadow: var(--token-elevation-higher-box-shadow);
+	display: flex;
+	padding: 16px;
+	width: 100%;
```

Comment:
The meaningful changes here are:

### Addition of `width: 100%` to `.root`

Fixes 🎟️ [[vqa] fix toast alignment](https://app.asana.com/0/1202097197789424/1202930794848928/f).

<img width="987" alt="CleanShot 2022-09-06 at 11 23 56@2x" src="https://user-images.githubusercontent.com/4624598/188674195-b44098ad-7bcf-4fe4-84df-ebc554021788.png">

### Removal of `padding-top: 2px` from `.contentArea`

Fixes minor alignment slash pixel-pushy issue.

<img width="1107" alt="CleanShot 2022-09-06 at 11 15 01@2x" src="https://user-images.githubusercontent.com/4624598/188673492-446f9471-1581-42b9-8131-4b5aff4cc647.png">

---

The remaining changes are formatting changes. Not sure why this formatting thrash is happening, but feels like something worth investigating post-launch 👍